### ISSUE TITLE
Suggested Binarization Speed Up?

### ISSUE THREAD

**Issue by GantMan** [atomic14/ar-browser-sudoku#4]
What do you think of this code for binarizing using tensors rather than JavaScript?

https://bit.ly/37a3bzm

**Comment by henrmota** [comment]
Also you can use tf.conv2d to apply the boxblur or gaussianfilter. By the way, amazing project :). One thing that I think that tensorflow doesn't receive enough credit it's image processing. An alternative way would be using something like webgl shaders, also a valid but more complicated approach.

https://codepen.io/henrmota/pen/yLazEGq?editors=0010

**Comment by GantMan** [comment]
WOW!  I didn't know a lot of this stuff.   Great work!

**Comment by henrmota** [comment]
Yes tensorflow is great for image processing. It doesn't receive enough appreciation for it.

**Comment by GantMan** [comment]
@henrmota - I have some questions about what you wrote.   Let me know if you'd be willing to chat TFJS sometime.   I'm on Twitter and I have a lot of slacks/discords for TFJS, but few people at our level.

**Comment by henrmota** [comment]
https://www.linkedin.com/in/henrmota83/. My twitter is https://twitter.com/choicept.

**Comment by henrmota** [comment]
@GantMan Hough line detection using tfjs. https://codepen.io/henrmota/pen/yLazEGq?editors=0010

**Comment by GantMan** [comment]
This is some epic stuff!   Connected with you on linkedin and twitter.

**Comment by cgreening** [comment]
Wow - I am way behind the curve - if you want to do a PR to add some of this stuff that would be amazing.
